source/features/wait-for-build.tsx

```diff
@@ -100,8 +100,8 @@ function init(): void {
 
 void features.add(__filebasename, {
 	asLongAs: [
-		// The user cannot merge
-		() => select.exists('[data-details-container=".js-merge-pr"]:not(:disabled)'),
+		// The user can merge
+		() => select.exists('[aria-label="Edit Pull Request title"]'),
```

Comment:
I'm confused about why we're checking for the title since that can also be edited by the PR owner. Can we use the sidebar instead? Or the merge button itself? Maybe they're both lazy-loaded though, so they can't be used.

Maybe this is just a soft-filter, which is ok, but the title needs to be updated anyway.